docs/modules/gear/consumables.mdx

```diff
@@ -0,0 +1,159 @@
+---
+id: consumables
+title: Custom Consumables
+---
+
+Custom consumables can be defined and applied to items in [kits](/docs/modules/gear/kits).
+When consumed, these items can trigger [actions](/docs/modules/mechanics/actions-triggers),
+and override vanilla eating or drinking behavior.
+
+### Consumables Element
+
+<div className="table-container">
+  <table>
+    <thead>
+      <tr>
+        <th>Element</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <label>{`<consumables> </consumables>`}</label>
+        </td>
+        <td>Node containing the custom consumable definitions.</td>
+      </tr>
+      <tr>
+        <th>Sub-elements</th>
+        <th />
+      </tr>
+      <tr>
+        <td>
+          <label>{`<consumable> </consumable>`}</label>
+        </td>
+        <td>A custom consumable definition.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+### Consumable Attributes
+
+<div className="table-container">
+  <table>
+    <thead>
+      <tr>
+        <th style={{ minWidth: "150px" }}>Attribute</th>
+        <th>Description</th>
+        <th style={{ minWidth: "100px" }}>Value</th>
+        <th style={{ minWidth: "100px" }}>Default</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <label>id</label>
+        </td>
+        <td>
+          <span className="badge badge--danger">Required</span>
+          Unique identifier used to reference this consumable from other places in the XML.
+        </td>
+        <td>
+          <span className="badge badge--primary">String</span>
+        </td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>
+          <label>action</label>
+        </td>
+        <td>
+          <span className="badge badge--danger">Required</span>
+          What happens when the consumable is consumed
+        </td>
+        <td>
+          <a href="/docs/modules/mechanics/actions-triggers">Action</a>
+        </td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>
+          <label>kit</label>
+        </td>
+        <td>
+          <span className="badge badge--danger">Alias</span>
+          Alternative to <label>action</label>, encouraged if the consumable applies a kit.
+        </td>
+        <td>
+          <a href="/docs/modules/gear/kits">Kit</a>
+        </td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>
+          <label>on</label>
+        </td>
+        <td>
+          How the consumable is consumed.
```

Comment:
```suggestion
        <td>
          <span className="badge badge--danger">Required</span>
          How the consumable is consumed.
```
Pretty sure this was made required, i'd also add a clarifying comment saying that while currently the only supported action is eat, in the future more actions (eg: clicking) may be supported